app/Http/Kernel.php

```diff
@@ -33,8 +33,9 @@ class Kernel extends HttpKernel
             \Coyote\Http\Middleware\FirewallBlacklist::class
         ],
         'api' => [
-            'throttle:60,1',
-            'bindings'
+            \Illuminate\Session\Middleware\StartSession::class,
+            \Illuminate\Routing\Middleware\SubstituteBindings::class,
+            Middleware\DefaultBindings::class,
```

Comment:
To throttle na pewno potrzebne? Ja wtedy dostaję (przy wejściu na api.4programmers.dev/users/1 na przykład):

![](http://i.imgur.com/ODkOYJw.png)